### ISSUE TITLE
Off-by-one error when setting a value

### ISSUE THREAD

**Issue by scribu** [tommoor/tinyrange-jquery-plugin#2]
I pre-set a value for the slider, like so:

```
<input type="range" step="5" max="200" min="5" value="80" name="radius">
```

It works fine on the first page load.

After I submit the form (without touching the slider), the slider's position is set to 85, instead of 80.

**Comment by tommoor** [comment]
Yep, I can reproduce this - will take a look and see if its easily fixed.
